uit/async_client.py

```diff
@@ -77,6 +101,8 @@ def __init__(
             delay_token=True,
         )
         self.env = AsyncHpcEnv(self)
+        self.scheduler = (None,)
+        self.commands = (None,)
```

Comment:
```suggestion
        self.scheduler = None
        self.commands = None
```
Should these really be tuples? These lines in uit.py are set to a plain None.

Reply:
So weirdly enough I had them set to plane old "None" before, but black was wanting to reformat them to use tuples.

Since it still worked and they get reassigned anyways I went with it, but I thought it was strange. I'm curious if it would suggest the same if the attributes were protected attributes instead of public. I think technically it should be just fine to set the to protected attributes instead since they're only referenced inside of Client.

Reply:
Well, I just changed it back and I'm still getting complaints when I run black locally: 
![image](https://github.com/user-attachments/assets/59f4a5b3-7ad7-4df1-9865-80c5d75a0a47)
The pipeline version doesn't seem to have any issues with it though. Not sure what that's about, but if the pipeline is happy, I'm happy.